index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Random book quotes</title>
+    <meta charset='utf-8' />
+  </head>
+  <body>
+    <p>THE LIST OF YOUR BOOKS</p>
+
+    <!--Add buttons to initiate auth sequence and sign out-->
+    <button id="authorize-button" style="display: none;">Authorize</button>
+    <button id="signout-button" style="display: none;">Sign Out</button>
+
+    <div id="content"></div>
+
+    <script type="text/javascript">
+      // Enter an API key from the Google API Console:
+      //   https://console.developers.google.com/apis/credentials
+      var apiKey = 'AIzaSyB-1fyO8l_CMTs31BkoFRKPS2bWQ18AwOc';
```

Comment:
Be aware using sensitive data, in a real projects you should never commit that data.